tools/build_journal.py

```diff
@@ -26,16 +26,24 @@ def main():
         help='Destination host for the payloads')
     parser.add_argument('--destaddr_as_host', action='store_true',
         help='Use destination address as the Host header')
+    parser.add_argument('--protocol', default=None,
+        help='Specify protocol: http or https (default http)')
```

Comment:
Is this true (default to http) or whatever is specified in the yaml files?

Reply:
Yup, we default the port to 80 in the YAML files.

Reply:
Right, but if the yaml file uses something else it won't be.
Personally I'd remove the default from the help.